tests/integration_tests/test_hyperopt_ray_horovod.py

```diff
@@ -71,8 +66,8 @@ def mock_storage_client(path):
             "lower": 0.001,
             "upper": 0.1,
         },
-        "combiner.num_fc_layers": {"space": "randint", "lower": 2, "upper": 6},
-        "combiner.num_steps": {"space": "grid_search", "values": [3, 4, 5]},
```

Comment:
num_steps is only for `tabnet` combiners, but we use concat in all of our test so this wasn't actually doing anything